assets/dev/js/editor/regions/panel/pages/menu/menu.js

```diff
@@ -73,6 +73,22 @@ PanelMenu.addAdminMenu = () => {
 		],
 	}, { at: 0 } );
 
+	PanelMenu.addItem( {
```

Comment:
@matipojo do you think this promotion should be on module or something?

Reply:
It's OK